eks-distro-base/check_update.sh

```diff
@@ -20,13 +20,12 @@ set -x
 
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 AWS_REGION="us-west-2"
-BASE_IMAGE=${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com/eks-distro/base:$(cat eks-distro-base/TAG_FILE)
-mkdir eks-distro-base/check-update
-cat << EOF >> eks-distro-base/check-update/Dockerfile
+BASE_IMAGE=${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com/eks-distro/base:$(cat TAG_FILE)
```

Comment:
Since we are already defining IMAGE_REPO and IMAGE_NAME in the Makefile, can we just pass that into the script or make use of it somehow so we don't redefine it in this bash script?

Reply:
Done